ejb/src/test/java/biz/karms/sinkit/ejb/impl/CoreServiceEJBTest.java

```diff
@@ -213,6 +273,36 @@ public IoCRecord getIoCForWhitelist(String ip, String fqdn, String sourceName, b
         return ioc;
     }
 
+    /**
+     * creates IoCRecord observed ageHours in the past
+     * @param ip
+     * @param fqdn
+     * @param sourceName feed name
+     * @param ageHours sets time.observation to current
+     * @return
+     */
+    public IoCRecord getIoCRecord(String ip, String fqdn, String sourceName, int ageHours, String classificationType) {
+        IoCRecord ioc = new IoCRecord();
```

Comment:
@hugeox If you are not going to overwrite the reference again, I suggest marking it final. It helps to immediately see some little bugs when one overwrites one's own variable unintentionally. It is not a requirement though, just a suggestion. The IoCRecord object is not immutable anyway...